src/ui/app.html

```diff
@@ -6,6 +6,13 @@
 		<meta name="viewport" content="width=device-width" />
 		%sveltekit.head%
 	</head>
+	<style>
+		html,
```

Comment:
I wonder if we should do some sort of CSS reset instead since we may have to deal with older browers?

Reply:
Stuff like [html conditional comments](https://www.quirksmode.org/css/condcom.html) to import polyfills / css compats?